docs/2_step_by_step/2_4_create_training_data.md

```diff
@@ -1 +1,102 @@
-# Create training data
\ No newline at end of file
+# Create training and evaluation (Optional) data
+
+Finetuner accepts [docarray](https://docarray.jina.ai/) `DocumentArray` as training data and evaluation data.
+Because Finetuner follows a [supervised-learning](https://en.wikipedia.org/wiki/Supervised_learning) scheme,
+you should assign label to each `Document` inside `DocumentArray` with:
```

Comment:
you should assign a label to each `Document` inside your `DocumentArray` as follows: